bigbluebutton-html5/imports/ui/components/audio/component.jsx

```diff
@@ -17,11 +17,7 @@ export default class Audio extends Component {
   }
 
   render() {
-    return (
-      <audio id="remote-media" autoPlay="autoplay">
-        <track kind="captions" /> {/* These captions are brought to you by eslint */}
-      </audio>
-    );
+    return null;
```

Comment:
I think we don't need this file anymore, since all it does is renders `null` and calls `init()` passed from the container.. in componentDidMount.

How about removing it and calling `init()` in the AudioContainer's componentDidMount?